markdown/bitburner.goopponent.md

```diff
@@ -8,5 +8,5 @@
 **Signature:**
 
 ```typescript
-type GoOpponent = "Netburners" | "Slum Snakes" | "The Black Hand" | "Tetrads" | "Daedalus" | "Illuminati";
+type GoOpponent = "Netburners" | "Slum Snakes" | "The Black Hand" | "Tetrads" | "Daedalus" | "Illuminati" | "????????????";
```

Comment:
Didn't you explicitly remove ????? from GoOpponent to fix a bug? Is this not just reintroducing the bug?

Reply:
This is just the visual documentation for the Go API, it is different than the core enum change you are thinking of. netscript definitions can't seem to import things from outside this file, so it's just a local type to the documentation.

I moved this and the other bugfixes to #1193

Reply:
This file can't import from outside this file, but the game can (and does) import types from here. In other words, this is the root of the types tree in many cases.

It was not immediately obvious that you weren't doing that here, especially when I was quickly checking on my phone XD